packages/nimble-components/src/rich-text/components/mention-view/mention-users-view/tests/rich-text-mention-users-view.spec.ts

```diff
@@ -0,0 +1,68 @@
+import { html } from '@microsoft/fast-element';
+import { RichTextMentionUsersView, richTextMentionUsersViewTag } from '..';
+import { type Fixture, fixture } from '../../../../../utilities/tests/fixture';
+import { waitForUpdatesAsync } from '../../../../../testing/async-helpers';
+
+async function setup(): Promise<Fixture<RichTextMentionUsersView>> {
+    return fixture<RichTextMentionUsersView>(
+        html`<nimble-rich-text-mention-users-view
+            mention-href="users:1"
+            mention-label="John Doe"
+            disable-editing
+            >@John Doe</nimble-rich-text-mention-users-view
+        >`
+    );
+}
+
+describe('RichTextMentionUsersView', () => {
+    let element: RichTextMentionUsersView;
+    let connect: () => Promise<void>;
+    let disconnect: () => Promise<void>;
+
+    beforeEach(async () => {
+        ({ element, connect, disconnect } = await setup());
+    });
+
+    afterEach(async () => {
+        await disconnect();
+    });
+
+    it('should export its tag', () => {
+        expect(richTextMentionUsersViewTag).toBe(
+            'nimble-rich-text-mention-users-view'
```

Comment:
fyi, all other usages should be the constant. This one stays as is since it is validating thee constant